pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -6322,7 +6322,6 @@ broken-packages:
   - incremental-computing
   - incremental-maps
   - increments
-  - indents
```

Comment:
unmark `indets` as broken as this package seems to be working correctly now. Unfortunetely we still can't use it with elm-format.

This can be excluded from the PR.